exporter/datadogexporter/examples/ootb-ec2.yaml

```diff
@@ -0,0 +1,109 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+       endpoint: "localhost:4318"
+      grpc:
+       endpoint: "localhost:4317"
+
+
+  hostmetrics:
+    collection_interval: 10s
+    scrapers:
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      cpu:
+        metrics:
+          system.cpu.utilization:
+            enabled: true
+          system.cpu.physical.count:
+            enabled: true
+          system.cpu.logical.count:
+            enabled: true
+          system.cpu.frequency:
+            enabled: true
+      disk:
+      filesystem:
+        metrics:
+          system.filesystem.utilization:
+            enabled: true
+      load:
+      memory:
+      network:
+      processes:
+
+  prometheus:
+    config:
+      scrape_configs:
+      - job_name: 'otelcol'
+        scrape_interval: 10s
+        static_configs:
+        - targets: ['0.0.0.0:8888']
+
+  filelog:
+    include_file_path: true
+    poll_interval: 500ms
+    include:
+      - /var/log/*/app.log
+    operators:
+      - type: json_parser
+      # Layout needs to match log timestamp format. If this section is removed, timestamp will correspond to time log was intake by DD.
+      - type: time_parser
+        parse_from: attributes.time
+        layout: '%Y-%m-%dT%H:%M:%S%z'
```

Comment:
Can we add comment, that for non-json logs user need to write a regex parser. or include parser https://github.com/open-telemetry/opentelemetry-helm-charts/blob/8319c0fcfdab578f94224fb6b7eefd33944702fa/charts/opentelemetry-collector/templates/_config.tpl#L216 commented ?